Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -220,13 +220,17 @@ def SwapKeepAndDrop(l):
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
 from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
 from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
+from Configuration.ProcessModifiers.mlpf_cff import mlpf
 from RecoLocalFastTime.Configuration.RecoLocalFastTime_EventContent_cff import *
 from RecoMTD.Configuration.RecoMTD_EventContent_cff import *
+from RecoParticleFlow.PFProducer.mlpf_EventContent_cff import *
 
 ctpps_2016.toModify(RECOEventContent, 
     outputCommands = RECOEventContent.outputCommands + RecoCTPPSRECO.outputCommands)
 phase2_hgcal.toModify(RECOEventContent,
     outputCommands = RECOEventContent.outputCommands + TICL_RECO.outputCommands)
+mlpf.toModify(RECOEventContent,
+    outputCommands = RECOEventContent.outputCommands + MLPF_RECO.outputCommands)
```

Comment:
please do not fill up the general EventContent_cff.py with area/subsystem-specific modifications.
edit `RecoParticleFlow_EventContent_cff` instead

Reply:
fixed in 4046d7731f2